config/sync/paragraphs.paragraphs_type.banner.yml

```diff
@@ -0,0 +1,18 @@
Y3A/MFPDRYTwWHlnKem8cgpuR3sg5AubuFGV+lbBOa17SHBSWFtKtKj9PdWpTIxG0ABFjX7hYAFPRpUjQ5pT5Q2JrANViBWl1FRUVMoyfpddTklN75Heyds4vGhUSafluqidE0CgRC2U0wjaSSpbQZnABYMMQRaMSkbQoc79LuWU3vkd7J2zi+qs+pTW0F3ZA6hA3SPDQSSrRNJK46HD2Vms1XNKnZ8ido5SdVw5v6XcqiKbkd7J2zhHZOJOyioVZxQrtc4qa0CEt0UfqTCaUKjlDlNGXplnbQCijiDTsptiptHKQ/NeN+Z+l3LKb3yO9k7XgKgyPf2TKJrTxARsoGaVuJWims/FINVDYy2T7pkVAEdigQLpO6tCd1XjOcp6XXDklNyO9k7BC4ZDsjqU0FWap3UYo1FFSOULgRqgwbhdk1wcVI2rqKIuAIJ0qpHBWp1FW8Xnk/odmKGQ7Jqpe7t7J2GQXBSGjSggCKFWcahNGifoiU93zKOXA4KOWoBRdUKMPbJU7EKd+FlAauWN2ABMB3JVpfWQoXi85SMn6Hct2yatL39vZOwuG+QKVAKLfXZQSxA/M5fFWcDrT7TCdno2mKm6fI0nRYlFacNKkr41iNrYe5QtEANdSULRBqdSUZHcN2FrtfCIJJKwlYSsBVKcz9LuWdimZH9vZBsEEBkOybugFRYQsIWELCFhVFhqovS7XKAWxFRf+PzuoZHtameh2GMVklxKOzemxOaGw18lTWeJ0EjGsAq1SRlj3NO4KJAQKYNVaWAAEKuQ5/0uQz0vdsmrte/t7INghlOybuVVVKBupkboQV6bNxrJGR2FFao38WtSR2TIZy6oZXXuhZZiQaAaoENbqV6qxrLW8t2Kt9pLXtbHqVZrS2Vu/zDso5PupwDDXmdncoMcdgnxvDa0Tcj+3sjY3kDSqEEn/qvh5PsvhpF8M9fDuT4nBpTTqVZrDPaehun3Vn9Hs0QBtEoJ+yt9nFntLmAfKdW3aKOyWiUjDH/APVD6MTQyvVvsws8tG9JGiAXolsZEx7JHUCk9VsTD14lJ6/GOiJS+uWl/SAE+3WuTqlKtU1GFzndlDG58nHLu+gKls+F3Fj0PcKz2hsgodHd0wEwkIihI5BTACsDU9owHktFSFDG0NU4HCcm5CKrB5WDysHlYPKweVHAXupiQ9N/d/qvw393+q/Df3f6r8N/d/qvw393+q/Df3f6r8O/d/qvw793+q/D/wB3/i+A/c/4vgP3P+L4D9z/AIp4ODT5q1QZUbrh+Vw/K4flYPKweVg8rB5WDysHlcPyuF5XC8rg/wCyjsxe4NDt/CHpP739U2MNAH2CIRCoiFRWzFwxRBek2l4xwB1MXSfKmjhZi487pJezR91bGPlsMEzmkOZo670iGzSMNWVeEQ5uEBtEWPJ1K9Ss/FgLxuxBOdhFVJawDQar46hoQo7TG/uqq1xGWJwCjDnNERdhcw6KZ7g0N3crFZcPzO3KapBSR3JYaOuf0O5LOsKNWn8pyau+U3hWXrQGmY3m+37sTduUN8liZWQH7IBUTmohFUVFbgeEEFA9zHNc00IKDjjjtUUPEqPmHlQQzSw2uSYBjHjRp7I6Er0y0cG0trs7Qp5k+4A+6Ar3JUmARuElAFOGsleGmoqpugqLqdXdFgkB0oU2DCakqC1uY7CdkyRjwrTYg842mhVmshx1kNSE0UCYdVahSS8ZimHRP6HfxyY+tqYrUaQuTN+SFZetDZG878i37sTdhyRlsPUf4QRRRRRutxpCo24k0UJCsfqE1lBDQCD91afU7TaGYDRrfFw0II3Ci9Ys7bMziavpsp/W5XaRNDQpLRNL1yONxFVaYCx2NqilxAkhMcHSlPDMQBG6ilMUtK6JrqhABVUZVqGzuU00KPQ7+OTH1tTFbPySm7nkhWXrQ2CN5ym+37sTdhyReLrBu+8oo32/ob/KgYpW4JKIIXE0RlqVxnJktd1XRMecRa5ENcFLZHaliwvYdjVRzVIa8Kb8xQE8MVVUCmlTisXLB/xu/jLRUuommhBUM+I6hWunBNCmb8jvdZetN2Ry2mV0UWIIW2Qr416ZapHvAvt+7E3p5IvF1gGjryijcQrcauY1WdtXAL1CPC5jk0hC6TpQCpcyr2UqtQSonlrhroinQscNQjY461QsrKglCjRRVCskQmmaw7blH06ThGWI1bU6J2rCLiqoGudp+VwVAqBUVFRUu0TSxMMWFWgtLflcmZTlsn5ibsjf2utgrA5De6zCsgvt36E3p5g2QVi6XXlFFFEq1Cto/gKzM7r1FtYa/ZNVUXIHEE+NzStRdECGFHcobi7Eqi43WG0tgkcXNqHCiNqiZZpeG+rS3Qd6phrVO0JuKbloVhPcIM+U/wAXDK863A0XFci+qZlohcLrJ+Ym7Iq0WgRD7lOtUpNQ6iZbpBvqm+oMHWpLRDJC6jxshuirKP8AIq3W79CZ08kXjZBWHZ2Qo3FTCsigFAFaW4oXjwmmiLk2Uuc4U2TXgGhKOqLGlCNoXZGNh3XDYNUXJzqhMxh2uouNw0QKadVLo68IXAVTIqoMDRQBEJ3Q7+M5YCuCPuuESQAV8E/7p9kc1pNU3kC6zGj01W2cxM03KdI5xq43Eotxbp/DwjDUFM2F1l6isQWNqtpBwJu15zC9q7qwnR2Qo3FTfmKIiiOoIUjcMjx5Wg3KlnDNGBcV5eHEqJ4ewFOrVYimv+6qnFAVVFUBVu3NwTXKXsbxcC0blMDCeoLUDRYysSd0u/jkx9QQU4/xuTdzyAgoOsKtGk+FaJnyOIcdAdFVyqVrdSoTboTQrEsQU5qQm7cgZAgrF3QuKcjfaBqCFE5DpVsbhtFfupwd+yDGkIsirShCsz8DuGduyKIUjMTdDqmYmtAN7nUWMppNdUDUKxNaZwXbDVWmAtxSPeA5x6bgUQXABPsM7Y8dBRCN5OjVHYiRVzqI2MDuvhIKagkqWxNArGSCo5nsdhejcT8rv4X/2Q=='
```

Comment:
I notice a lot of binary data here but as far as I can tell from the source of the YAML file it is ~70KB which is within the limits suggested by @rasben .